src/MICore/MIResults.cs

```diff
@@ -31,7 +31,7 @@ public enum ResultClass
         exit
     }
 
-    public class ResultValue
+    public abstract class ResultValue
```

Comment:
I originally wrote it so that I could compose the MI record again. The ToString implementations did not adhere to the MI result format, so I created a separate property to return what I needed at the time. Eventually, the consume was rewritten so that I didn't need the properly formatted record anymore and forgot to remove the implementation. I'll remove it.